### PR TITLE
Guidelines for including new qMRI suffixes

### PR DESCRIPTION
**Pull request by Gilles86**
Following issue #23

### CODE REVIEW

**Comment by ChristophePhillips** [comment]
Looks good overall.

I guess the wording "If the qMRI map is generated using open-source software:" will addressed in a separate pull request.

File: src/99-appendices/10-qmri.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ magnetization prepared two gradient echoes (MP2RAGE). Such generic acquisitions
 can serve as a basis to derive various qMRI applications by changes to
 the acquisition sequence (e.g. readout) type or varying additional scan parameters.
 
-If such inheritance relationship is applicable between an already existing
+If such a inheritance relationship is applicable between an already existing
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:40]: an

File: src/99-appendices/10-qmri.md [hunk 2]
```diff
@@ -92,14 +92,43 @@ For a dataset with a `grouping suffix`, the BIDS validation is successful if:
...
+
+* All grouping suffixes MUST be capitalized.
+* Grouping suffixes MUST attain a clear description of the qMRI application
+that they relate to. Hyperlinks to example applications and/or more detailed
+descriptions are encouraged whenever possible.
+* Unless the pulse sequence is exclusively associated with a specific qMRI
+application (e.g. `MP2RAGE`), sequence names are NOT used as grouping suffixes.
+* If it is possible to derive a qMRI application from an already existing
+grouping suffix by defining a set of logical conditions over the metadata
+fields, the [table of method-specific priority levels](#prioritylevels) and the
```
Comment (agahkarakuzu) [inline src/99-appendices/10-qmri.md:111]: These hyperlinks may have changed, we should double check during proofreading.

**Comment by agahkarakuzu** [comment]
> I guess the wording "If the qMRI map is generated using open-source software:" will addressed in a separate pull request.

@ChristophePhillips good idea, doing that after merging this PR is also good not to deal with merge conflicts.

**Pull request merged**